api/v1alpha2/types.go

```diff
@@ -96,6 +96,10 @@ type NetworkSpec struct {
 	// Subnets configuration.
 	// +optional
 	Subnets Subnets `json:"subnets,omitempty"`
+
+	// Allow for configuration of load balancer backend (useful for changing apiserver port)
+	// +optional
+	LoadBalancerBackendPort *int64 `json:"loadBalancerBackendPort,omitempty"`
```

Comment:
Any specific reason for `int64` here instead of `int32`?

Reply:
Yeah, all of the GCP compute services seem to require int64 for ports. So I went ahead and just used int64 directly here instead of converting it later on. We convert the Cluster.Spec.ClusterNetwork.APIServerPort field in the LoadBalancerFrontendPort function as well. I could change it to int32 though, I suppose if it makes more sense to have it match Cluster.Spec.ClusterNetwork.APIServerPort